hexa/datasets/schema/mutations.py

```diff
@@ -178,6 +178,33 @@ def resolve_delete_dataset_share(_, info, **kwargs):
         return {"success": False, "errors": ["PERMISSION_DENIED"]}
 
 
+@mutations.field("generateUploadUrl")
+def resolve_generate_upload_url(_, info, **kwargs):
```

Comment:
I'm pretty sure there is a permission that need to be checked here ("update" or something like that)

Reply:
@nazarfil Did you address this ?